docs/sources/menu.yaml

```diff
@@ -199,6 +208,13 @@
           name: Stat
         - link: /panels/visualizations/table-panel/
           name: Table
+          children:
+          - name: Overview
+            link: /panels/table/
+          - link: /panels/table/table-field-options/
```

Comment:
Missing word visualizations

Reply:
That should have thrown an error. WTH. Working on the fix now. Thank you for spotting these!

Reply:
@robbymilo is hopefully going to make menu.yaml go away in the not-distant future, so probably isn't worth figuring out why it's not giving me errors.

Reply:
Menu fix pushed.